velox/functions/sparksql/tests/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ add_executable(
   velox_functions_spark_test
   ArithmeticTest.cpp
   ArraySortTest.cpp
+  ArrayMaxTest.cpp
```

Comment:
When possible, try to preserve alphabetical order.

Reply:
fix